contents/blog/the-posthog-array-1-34-0.md

```diff
@@ -0,0 +1,98 @@
+
+Best of all, if your team uses [Session Recording](/product/session-recording), we'll also add your most recent recordings to the homepage too. We've found users who watch these recordings make 3x more useful discoveries, on average!
+
+### New: Data Management
+![PostHog - view definition context](../images/blog/data-management-feature/data_management_view.gif)
+
+We've also added a new Data Management system! This new tool  helps you organize your PostHog data in-house through event definitions, definition tagging and many more features. This is especially useful in large teams, where it can sometimes be difficult to know if your insights are using the right events. 
+
+Check out our [blogpost which explains more about Data Management on PostHog](/blog/data-management-feature) or dive straight into the [Data Management docs](https://posthog.com/docs/user-guides/data-management) for more information.
+
+> 🎁 Definition creation, tagging, and volume metrics are premium features which require a PostHog Scale or Enterprise license. [Learn more](/pricing).
+
+### Improvement: Smart insight naming
+![PostHog - smart insight naming](../images/blog/array/1_34_0-smart-naming.png)
+
+Previously, we generated random default names for insights based on animals. It was fun and ensured each insight had a unique name, but they weren't very descriptive and didn't help if you wanted to browse existing insights. 
+
+Now, we've added smart insight naming to automatically generate descriptive names for every insight you create. This makes it a _lot_ easier to find the insights you need - thought you can of course add custom animal names, if you want.
+
+### New: Property group filtering
+One of the most requested features for PostHog, property group filters enable you to mix and match your favourite property filters with ```OR```s and ```AND``` conditions. This makes it much simpler to create complex filters when building insights, though this is currently only available as a global filter for insights.
+
+### New: Staff user controls
+Self-hosted users can now manage staff users on their instance settings. Staff users are a special instance-level permission which have access to instance-wide settings - and you probably want to have multiple staff users enabled to ensure your instance can always be properly maintained.
+
+You can manage staff users either via the PostHog UI, or the API if you prefer - check out [the staff user docs](//docs/self-host/configure/instance-settings#staff-users) for more information.
+
+### Other improvements & fixes
+Version 1.34.0 also adds hundreds of other fixes and improvements, including...
+
+- **Improved**: Dashboard "add insight" flow
+- **Improved**: SSO login redirect
+- **New:** Experiments secondary metrics
+- **New**: PostHog storybook
+- **New**: History logging
```

Comment:
I don't think this is out for self-hosted? @pauldambra

Reply:
```suggestion
```